server/app.js

```diff
@@ -40,6 +40,9 @@ io.on("connection", (socket) => {
     const verifyToken = jwt.verify(token, process.env.JWT_SECRET);
     socket.tokenId = verifyToken.id;
     console.log(`connected by ID of ${socket.tokenId}`);
+    socket.on("send-message", (message, otherUserId) => {
+      socket.emit("receive-message", message);
```

Comment:
Won't this send a message to all the connected clients? Since we are using private messaging, you need to send the message through an individual socket id.